### PR TITLE
[#120455] Readme for Split Accounts engine

### PR DESCRIPTION
**Pull request by jhanggi**
@benlinton Any suggestions of other things to add?

### CODE REVIEW

**Comment by jhanggi** [comment]
@benlinton could I get a look over this?

File: vendor/engines/split_accounts/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Make sure the `gem "split_accounts",   "~> 0.0.1", path: "vendor/engines/split_accounts"`
+line is enabled in `Gemfile`. Then ensure the `feature.split_accounts_on` in
+`Settings.yml` is set to `true`.
+
+## Notes
+
+* Only global admins may create a Split Account. Other account admins and facility
+    admins may edit the membership list.
+* `SplitAccounts` expire when the earliest of its children expire
```
Comment (benlinton) [inline vendor/engines/split_accounts/README.md]: Missing `.`

File: vendor/engines/split_accounts/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+* If you suspend a child account, the parent gets suspended as well. However, if you
+    unsuspend a child account, the parent remains suspended. You must re-activate
+    the parent account yourself.
+
+## "Apply Remainder" Behavior
+
+The costs are calculated by multiplying the cost of an order by the percentage of
+each split. The result is then rounded down to the penny. We then add up each
+accounts' amount; if there is a difference with the original amount, the difference
+is applied to the account that has "Apply Remainder" selected.
```
Comment (benlinton) [inline vendor/engines/split_accounts/README.md]: Maybe mention: (`extra_penny`)

File: vendor/engines/split_accounts/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    * C: $3.49
+
+If the "Apply Remainder" flag were set on on account B, the results would be slightly
+more evenly:
+
+    * A: $3.51
+    * B: $3.50
+    * C: $3.49
+
+
```
Comment (benlinton) [inline vendor/engines/split_accounts/README.md]: Extra newline.

**Comment by benlinton** [comment]
:+1:

File: vendor/engines/split_accounts/README.md (hunk 2 above)
Comment (jhanggi) [inline vendor/engines/split_accounts/README.md]: I've completely renamed "extra penny" "apply remainder" even down at the database level.

**Pull request merged**